unlock-wordpress-plugin/inc/classes/blocks/class-unlock-box-block.php

```diff
@@ -43,8 +43,14 @@ protected function setup_hooks() {
 		 * Actions.
 		 */
 		add_action( 'init', array( $this, 'register_block_type' ) );
+	
+		add_action('add_meta_boxes', array( $this, 'unlock_pv2_add_custom_meta_box' ) );
```

Comment:
Please do not use `pv2` here because it is unclear what this means...

Reply:
Noted

Reply:
same below BTW :)